hm_3.py

```diff
@@ -0,0 +1,4 @@
+mas_list_numbers = [ i for i in range(0,99999999999)]
```

Comment:
This line works too slow in Python 2.X.
According to task description, your code should work under both Python 2.X and Python 3.X.